docs/source/code.rst

```diff
@@ -0,0 +1,61 @@
+Code Guidelines
+-------------------------------------------
+
+Developers of eMach should read this document to understand expectations of code contributed to the eMach codebase. This document outlines the 
+code requirements for ``analyzers`` and ``machine_designs``. Understanding of proper Python coding practices are necessary to contribute to each 
+section of ``eMach``.
+
+Analyzer Code
+++++++++++++++++++++++++++++++++++++++++++++
+
+The code required for each ``analyzer`` within the ``eMach`` repository must be structured to analyze any set of user-defined inputs and produce 
+user-defined outputs. The code structure of each ``analyzer`` should contain each of the following classes:
+
+1. DesignProblem Class
+2. DesignAnalyzer Class
+
+DesignProbelm Class
+*******************************************
+
+The ``DesignProblem`` class takes all of the user-defined inputs as its arguments. The output of the ``DesignProblem`` class is a single variable with 
```

Comment:
> The ``DesignProblem`` class takes all of the user-defined inputs as its arguments.

I think you mean to be talking about the initializer for the class. But this sentence isn't particularly helpful. I mean... this is what an initializer does for any class ever written: it takes user-defined inputs as arguments

Instead, I think you want to be telling us what type of information a developer is expected to have the problem class's initializer take as an input.